### DISCUSSION TITLE
Classic mod mania

### DISCUSSION THREAD

**Discussion by KoriVR** [ppy/osu#22713] [Ideas]
Hello! When was classic mod on mania removed and why?
I usually use classic mod when making plays on mania.

**Comment by theredpro-sj** [comment]
I see no reason why there shouldn't be one as it is on the leaderboard ui's.

**Reply by Tenexxt** [reply]
There isn't a Classic Mod for mania because it wouldn't be able to do anything. All the differences in gameplay from stable to lazer are [here](https://osu.ppy.sh/wiki/en/Client/Release_stream/Lazer/Gameplay_differences_in_osu%21%28lazer%29), and on all of them for mania its said that these changes aren't revertable using a mod, so this wouldn't work (unless the page wasn't updated for a long time and its possible now, but i doubt it)

The Classic Mod exists on UI's to signal which scores are from stable and which ones are from lazer
